modules/end-user-guide/pages/using-a-git-provider-access-token.adoc

```diff
@@ -116,6 +116,20 @@ __<Secret_prepared_in_step_5>__
 EOF
 ----
 ====
+. Azure DevOps Server. If you are using Azure DevOps Server, you must also modify the xref:mounting-git-configuration.adoc[workspace's
```

Comment:
@vinokurig could you please make this block `IMPORTANT` similar to https://github.com/eclipse-che/che-docs/pull/2851/files#diff-ac7779248c50ba9a999fd70fb4eb27a90ee942e6709da76b96a988ff6b548ce1R39